conf/config.json

```diff
@@ -272,7 +272,14 @@
             "coldStorageArchiveTopicPrefix": "cold-archive-req-",
             "coldStorageRestoreTopicPrefix": "cold-restore-req-",
             "coldStorageGCTopicPrefix": "cold-gc-req-",
-            "coldStorageStatusTopicPrefix": "cold-status-"
+            "coldStorageStatusTopicPrefix": "cold-status-",
+            "supportedLifecycleRules": [
```

Comment:
not needed, this is the default (and the default should generally be used, to ensure we don't miss new features)